examples/fruitjam_ntp_settings.toml

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Tim Cocks for Adafruit Industries
```

Comment:
I also just noticed all the copyright notices are set to my name. You can set these to your name or handle if you want.